src/macaron/malware_analyzer/pypi_heuristics/metadata/wheel_absence.py

```diff
@@ -120,10 +121,9 @@ def analyze(self, pypi_package_json: PyPIPackageJsonAsset) -> tuple[HeuristicRes
             )
 
             # use a head request because we don't care about the response contents
-            if send_head_http_raw(inspector_link) is None:
-                inspector_links.append(None)
-            else:
-                inspector_links.append(inspector_link)
+            inspector_links[inspector_link] = False
```

Comment:
Is it possible for `inspector_link` value to be `None` or empty? If yes, can you please add a check before adding it to the dictionary?